workers/unity/Assets/Playground/Scripts/Bootstrap.cs

```diff
@@ -31,9 +28,9 @@ public void Awake()
             PlayerLoopManager.RegisterDomainUnload(DomainUnloadShutdown, 10000); // Clean up worlds and player loop
 
             Application.targetFrameRate = TargetFrameRate;
+#if UNITY_EDITOR
```

Comment:
Doesn't this have to be inside of the if statement? Wont the code be invalid if UNITY_EDITOR is false?